pkg/cloudprovider/aws/suite_test.go

```diff
@@ -121,6 +121,20 @@ var _ = Describe("Allocation", func() {
 
 	Context("Reconciliation", func() {
 		Context("Specialized Hardware", func() {
+			It("should not launch AWS Pod ENI on a t3", func() {
```

Comment:
Can you have a positive test that shows that it does schedule w/ this requirement?

Reply:
Done!